neo/IO/Data/Slice.cs

```diff
@@ -1,21 +1,37 @@
 using Neo.Cryptography;
 using System;
 using System.Linq;
+using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 using System.Text;
 
-namespace Neo.IO.Data.LevelDB
+namespace Neo.IO.Data
```

Comment:
This class is only used in leveldb, right?

Reply:
Yes, I can move it back to LevelDB